php_phongo.c

```diff
@@ -45,13 +45,15 @@
 #include <Zend/zend_exceptions.h>
 #include <ext/spl/spl_iterators.h>
 #include <ext/spl/spl_exceptions.h>
-/* For formating timestamp in the log */
-#include <ext/date/php_date.h>
 /* Stream wrapper */
 #include <main/php_streams.h>
 #include <main/php_network.h>
 /* Debug log writing */
 #include <main/php_open_temporary_file.h>
+/* For formating timestamp in the log */
+#include <ext/date/php_date.h>
+/* String manipulation */
+#include <Zend/zend_string.h>
```

Comment:
Should this be conditional for [PHP 7](http://lxr.php.net/xref/PHP_MASTER/Zend/zend_string.h) or [PHP 5.x](http://lxr.php.net/xref/PHP_5_6/Zend/zend_string.h)? The implementations are quite different.